modules/thermal_hydraulics/src/utils/MeshAlignment1D3D.C

```diff
@@ -0,0 +1,139 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/primary/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "MeshAlignment1D3D.h"
+#include "Assembly.h"
+#include "KDTree.h"
+
+#include "libmesh/elem.h"
+
+MeshAlignment1D3D::MeshAlignment1D3D(const MooseMesh & mesh) : MeshAlignmentBase(mesh) {}
+
+void
+MeshAlignment1D3D::initialize(
```

Comment:
It apparently does not compile because of some dumb inheritance rule with overloaded methods. Basically, I'd need to rename either the `initialize` for 1D-2D or 2D-2D.